### ISSUE TITLE
hash added after table name automatically

### ISSUE THREAD

**Issue by aihornmac** [rethinkdb/horizon#482]
on client side, `horizon('_demo_search').watch().subscribe(...)` throws Error

```
Table `newa._demo_search_2e35109b04ee` does not exist in: r.db("newa").table("_demo_search_2e35109b04ee").wait({"waitFor": "all_replicas_ready"}) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
```

I wonder why hash is added automatically, and how to prevent this.
Plz help!

@horizon/client 1.0.2
@horizon/server 1.0.2

**Comment by deontologician** [comment]
Hi @aihornmac, 
Internally, horizon renames the tables to avoid a race condition. You shouldn't create the tables directly in rethinkdb for the time being, you need to create them either by starting the server with `--auto-create-collections=true` or by having a schema and using `hz set-schema` to create it.

Although it's not causing your problem, we do plan to get rid of the random string appended to the table name so that it's easier to access horizon tables from the dataexplorer etc.

**Comment by dalanmiller** [comment]
Since this is the expected result for the moment, I'm going to close this issue. @aihornmac if you have a proposal to make this or `set-schema` better feel free to open a new issue.

**Comment by deontologician** [comment]
Do we have an issue for fixing this? If not we could use this one

**Comment by deontologician** [comment]
This was resolved in Horizon 2.0.0, we no longer add hashes to the end of table names
